composer.json

```
@@ -19,6 +19,7 @@
     "spryker/oms": "^7.0.0 || ^8.0.0 || ^10.0.0 || ^11.0.0",
     "spryker/propel-orm": "^1.5.0",
     "spryker/quote": "^1.0.0 || ^2.0.0",
+    "spryker/router": "^1.11",
```

There is no such version like `^1.11"`
 
 ```suggestion
    "spryker/router": "^1.1.0",
```

---

it was added automatically by composer require

---

When we need to add some dependency to the module, we usually should use the lowest minor (where our code is available) of the latest major, in our case it's ^1.1.0
See https://github.com/spryker/router/releases/tag/1.1.0